beacon_node/beacon_chain/Cargo.toml

```diff
@@ -19,6 +19,7 @@ environment = { workspace = true }
 serde_json = { workspace = true }
 
 [dependencies]
+clap = { workspace = true }
```

Comment:
Is it worth to concern the beacon-chain with ArgMatches? `GraffitiOrigin` could take some enum that the cli crate parses

Reply:
yeah good call - i moved this processing back to the CLI crate